initial/src/main/java/com/example/resources/CourseResource.java

```diff
@@ -0,0 +1,26 @@
+package com.example.resources;
+
+import com.example.services.CourseService;
+import com.fasterxml.jackson.core.JsonProcessingException;
+import org.springframework.beans.factory.annotation.Autowired;
+import org.springframework.web.bind.annotation.PathVariable;
+import org.springframework.web.bind.annotation.RequestMapping;
+import org.springframework.web.bind.annotation.RestController;
+
+@RestController
+public class CourseResource {
```

Comment:
Please add *Javadoc* to explain
1. What is a course
2. What is this resource about

Reply:
I added a javadoc line here, though I am not sure what else to put that would be unique from the getCourseData() javadoc, will be curious to hear your thoughts on what I included